easybuild/easyconfigs/c/CPLEX/CPLEX-12.6.3.eb

```diff
@@ -0,0 +1,18 @@
+name = 'CPLEX'
+version = '12.6.3'
+
+homepage = 'http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/'
+description = """IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency,
+ reducing costs, and increasing profitability."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['cplex_studio%s.linux-x86-64.bin' % ''.join(version.split('.'))]
+
+dependencies = [
+    ('Java', '1.8.0_66', ''),
+    ('foss', '2015b', ''),
```

Comment:
When solving a MIP with distributed parallel optimization, MPI is one of the solutions offered by CPLEX. So either, we add foss as a dependency/toolchain, or require the user to load it him/herself. I'll leave it up to you if foss should be used as a toolchain instead. The same remarks holds for adding Java.